subiquity/common/types/__init__.py

```diff
@@ -91,6 +91,8 @@ class ApplicationState(enum.Enum):
     UU_RUNNING = enum.auto()
     UU_CANCELLING = enum.auto()
```

Comment:
I wonder if we want to drop this as well.

Reply:
@d-loose Do you see a problem with removing one this value from the enumeration?

Reply:
Not at all, we're not using it anywhere in the desktop installer